.github/actions/with-k-docker/Dockerfile

```diff
@@ -15,9 +16,13 @@ RUN    apt-get -y update    \
          make               \
          python3.10-dev     \
          python3.10-venv    \
-         /llvm-backend.deb  \
-         /kframework.deb    \
-    && apt-get -y clean
+         /kframework.deb
+
+RUN if [ "${INSTALL_BACKEND_DEBS}" = "true" ]; then \
+      apt-get -y install /llvm-backend.deb; \
+    fi
```

Comment:
We should clean this up a bit (in a different PR): the `LLVM_BACKEND_DEB_PATH` is required, even if `INSTALL_BACKEND_DEBS` is not `true`.